iOS_4x_release.md

```diff
@@ -0,0 +1,42 @@
+# Aligning major version releases for the Adobe iOS SDK
+
+### Why are we getting another major update already?
+
+Apple has started doing an annual requirement to increase the minimum version of Xcode used when submitting to the App Store.  Each Xcode version only supports back to a specific minimum version of iOS.  Loosly speaking, Xcode version and _minimum_ iOS version are correlated.  
+
+https://developer.apple.com/ios/submit/
+
+In order to align with Apple's changing requirements, Adobe will also be doing an annual major release of its iOS SDKs.  In this yearly major update, you can expect the following changes:
+  - Increasing the minimum supported version of iOS.
+  - Increasing the minimum supported version of Xcode.
+  - Possible increase to minimum supported version of Swift.
+
+### Why are all the extensions updating to major version 4?
+
+The main goal is to simplify the Adobe SDK distibution.  Aliging all extesions on the same major version allows us to create predictability.  For example:
```

Comment:
"Aligning the extensions on the same major version..."

Also, a double space between "... to create predictability." and "For example:".